context-free.md

```diff
@@ -2,21 +2,55 @@
 
 ##  Who is this programming language for?
 
+Context Free is useful for people who are interested in creating
+computer-generated art.  This could have uses for graphical designs or just
+expressing creativity.  Other features in Context Free could interest animators
+as well.  
 
 ## What is easy to do in this language? Why is it easy?
 
+In Context Free, it is very easy to draw art recursively.  This allows the
+creator to make fractals and other patterns with relatively little work.  This
+is made possible since shapes can include themselves in their own description, 
+so Context Free can make complicated images that would be very difficult to
+draw.  In addition, Context Free is good at incorporating randomness in images.
```

Comment:
Sure thing!  In Context Free you can include multiple definitions of a shape and (optionally) assign a weight to each.  Then, when the shape is called, Context Free will randomly choose one the definitions to render based on the weight.  You can also use random numbers as inputs to shape adjustments in theory (but I couldn't figure out how to do this in version 2.2).  [Jeb](https://github.com/hmc-cs-jbearer/what-is-a-DSL/blob/b0738d90c38acad329975ada6337f0ecb7a3e4fd/jeb_bearer_hzl.png), [Woky](https://github.com/wpark/what-is-a-DSL/blob/2e5beccfb641c985113407e25689d95a83cc7ebb/woky_park_LNR.png), and [Prof Ben](https://github.com/bwiedermann/what-is-a-DSL-fall16/blob/c318a8712db133aaea3dee9d588213b2f7f41c12/ben_wiedermann_nan.png) all used randomness with great effect in their programs.